root/server/gettext/poFile.mjs

```diff
@@ -6,22 +6,24 @@
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
-/* eslint-disable import/no-commonjs */
+import fs from 'fs';
+import path from 'path';
 
-const fs = require('fs');
-const path = require('path');
+import po2json from 'po2json';
 
-const po2json = require('po2json');
+import {
+  MB_SERVER_ROOT,
+} from '../../static/scripts/common/DBDefs.js';
 
 const LOCALE_EXT = new RegExp('_[a-zA-Z]+\\.po$');
-const PO_DIR = path.resolve(__dirname, '../../../po');
+const PO_DIR = path.resolve(MB_SERVER_ROOT, 'po');
 
-exports.path = function (domain, locale) {
+export function getPath(domain, locale) {
```

Comment:
I'm changing the name because `path` is already bound (as the name of an import).  I'm guessing this is why it was previously added to the exports object.